app/templates/components/ui-accordion.hbs

```diff
@@ -0,0 +1 @@
+{{yield}}
```

Comment:
This template can be removed since you're anyway extending the original accordian